apps/server/test/degrees/[DELETE].test.ts

```diff
@@ -0,0 +1,50 @@
+import request from 'supertest'
+import { getApp } from 'app'
+import type { Express } from 'express'
+import { getSource } from '@modtree/typeorm-config'
+import { oneUp } from '@modtree/utils'
+import { setup, teardown } from '@modtree/test-env'
+import { Api } from '@modtree/repo-api'
+
+const dbName = oneUp(__filename)
+const db = getSource(dbName)
+let app: Express
+let api: Api
+let findOneById: jest.SpyInstance
+let remove: jest.SpyInstance
+
```

Comment:
what's the difference between this test and `/degrees/:degreeId/[DELETE].test.ts`